src/ontology/mondo-edit.obo

```diff
@@ -371316,9 +371316,11 @@ xref: Orphanet:94063 {source="MONDO:equivalentTo"}
 xref: SCTID:719046005 {source="MONDO:equivalentTo"}
 xref: UMLS:C4305140 {source="MONDO:equivalentTo"}
 xref: UMLS:CN206727 {source="MONDO:equivalentTo"}
-is_a: MONDO:0000508 {source="Orphanet:94063"} ! syndromic intellectual disability
+is_a: MONDO:0000761 {source="Orphanet:94063"} ! syndrome caused by partial chromosomal deletion
+is_a: MONDO:0015160 {source="https://www.rarechromo.org/media/information/Chromosome%2012/12q14%20microdeletions%20FTNW.pdf"} ! multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
 is_a: MONDO:0016877 {source="Orphanet:94063"} ! partial deletion of the long arm of chromosome 12
-is_a: MONDO:0019703 {source="Orphanet:94063", source="Orphanet:94063/inferred"} ! primary bone dysplasia with increased bone density
+relationship: excluded_subClassOf MONDO:0000508 {source="Orphanet:94063"} ! syndromic intellectual disability
+relationship: excluded_subClassOf MONDO:0019703 {source="Orphanet:94063", comment="Orphanet:94063/inferred"} ! primary bone dysplasia with increased bone density
```

Comment:
this should be source, not comment 
(an easy mistake because Protege defaults to the comment annotation property)